app/plugins/Files/js/loaders.js

```diff
@@ -0,0 +1,89 @@
+'use strict';
+
+// TODO: Can a webview use IPC.sendSync?
+function download(nickname) {
+	// TODO: setup dialog system
+	var savePath = IPC.sendSync('dialog', 'save', {
+		defaultPath: nickname,
+	});
+	if (!savePath) {
+		return;
+	}
+	notify('Downloading ' + nickname + ' to '+ savePath +' folder', 'download');
+	IPC.sendToHost('api-call', {
+		url: '/renter/files/download',
+		type: 'GET',
+		args: {
+			nickname: nickname,
+			destination: savePath
```

Comment:
try to be consistent about lines ending in `,`. Go style enforces `,` on every line of a struct, so I'd prefer to use that style in the UI as well if it makes sense.